demisto_sdk/commands/format/format_module.py

```diff
@@ -140,10 +144,15 @@ def format_manager(
         add_tests (bool): Whether to exclude tests automatically.
         id_set_path (str): The path of the id_set.json file.
         clear_cache (bool): wether to clear the cache
+        format_with_graph (bool): wheter to use the graph in format
     Returns:
         int 0 in case of success 1 otherwise
     """
 
+    if id_set_path:
+        logger.error('The flag --id-set-path was added to the command, but the usage of the id_set is deprecated.'
+                     ' demisto-sdk format now uses the content graph.')
```

Comment:
Do we surely know all SDK users have all dependencies for using the graph? @dantavori